lib/ui/compositing.dart

```diff
@@ -235,11 +235,13 @@ class SceneBuilder extends NativeFieldWrapperClass2 {
   /// Adds a backend texture to the scene.
   ///
   /// The texture is scaled to the given size and rasterized at the given offset.
-  void addTexture(int textureId, { Offset offset: Offset.zero, double width: 0.0, double height: 0.0 }) {
+  ///
+  /// If `freeze` is true new frames will not be consumed from the texture.
```

Comment:
Can this docstring be more detailed please?

Reply:
Sure, can you help me by explaining what wasn't clear when you read this?

Reply:
* Who not consume the textures?
* When should the flag be set?

Reply:
Thanks, rewrote the comment PTAL.